code/modules/atmospherics/gasmixtures/gas_mixture.dm

```diff
@@ -9,8 +9,8 @@ What are the archived variables for?
 #define SPECIFIC_HEAT_N2O		40
 #define SPECIFIC_HEAT_AGENT_B	300
 
-#define HEAT_CAPACITY_CALCULATION(oxygen, carbon_dioxide, nitrogen, toxins, sleeping_agent, agent_b) \
-	(carbon_dioxide * SPECIFIC_HEAT_CDO + (oxygen + nitrogen) * SPECIFIC_HEAT_AIR + toxins * SPECIFIC_HEAT_TOXIN + sleeping_agent * SPECIFIC_HEAT_N2O + agent_b * SPECIFIC_HEAT_AGENT_B)
+#define HEAT_CAPACITY_CALCULATION(oxygen, carbon_dioxide, nitrogen, toxins, sleeping_agent, agent_b, innate_heat_capacity) \
+	(carbon_dioxide * SPECIFIC_HEAT_CDO + (oxygen + nitrogen) * SPECIFIC_HEAT_AIR + toxins * SPECIFIC_HEAT_TOXIN + sleeping_agent * SPECIFIC_HEAT_N2O + agent_b * SPECIFIC_HEAT_AGENT_B + innate_heat_capacity)
```

Comment:
Assume making this equation an FFI call has more overhead than doing it in BYOND?

Reply:
It's more that I haven't moved tanks into MILLA yet, so we still need to do gas_mixture calculations for, e.g., TTV bombs.